models/demos/falcon7b/tt/falcon_attention.py

```diff
@@ -238,7 +238,7 @@ def forward(
         ###########
         ### TMs ###
         ###########
-        query_layer, key_layer, value_layer = tt_lib.tensor.nlp_create_qkv_heads(
+        query_layer, key_layer, value_layer = tt_lib.tensor.nlp_create_qkv_heads_falcon7b(
```

Comment:
@TT-BrianLiu and @arakhmati can Falcon we uplifted to use ttnn APIs ?
It's the same OP/kernels under the hood, right?

Reply:
I had to change the cb size for the generic implementation. For falcon7b, since head_dim is small, I hard-coded the cb size to fit the entire W dim (double buffered, this is same as BERT i think). For the generic one, I don't base this cb off of input size. It's tiny now, so perf may be worse.

Reply:
We should measure and if perf is a lot worse for general shapes, we can optimize the general one further